Common/Constants.cs

```diff
@@ -84,8 +84,6 @@ public static TimeSpan MaxAllowedExecutionTime
 
         // TODO: need localization?
         public const string SiteExtensionProvisioningStateNotFoundMessageFormat = "'{0}' not found.";
```

Comment:
SiteExtensionProvisioningStateNotFoundMessageFormat  is still using in [SiteExtensionManager.cs](https://github.com/projectkudu/kudu/pull/1992/files#diff-afacb573b03286d1f84a90cda5b57ddaR468)